docs/src/using-parsers/7-abi-versions.md

```diff
@@ -14,7 +15,13 @@ A given version of the tree-sitter library is only able to load parsers which ha
 | >=0.20.3, <=0.24    | 13                     | 14                     |
 | >=0.25              | 13                     | 15                     |
 
-By default, the tree-sitter CLI will generate parsers using the latest available ABI for that version. Grammar authors can specify an older ABI (within the constraints _of the CLI_, which may be stricter than the library!) via the `--abi` option to the `generate` command:
+By default, the tree-sitter CLI will generate parsers using the latest available ABI for that version. Grammar authors can
+specify an older ABI (within the constraints _of the CLI_, which may be stricter than the library!) via the `--abi` option
+to the `generate` command:
+
 ```
 tree-sitter generate --abi=<DESIRED-ABI>
 ```
+
+Note that while the CLI doesn't support generation for ABIs prior to 14, existing ABI 13 parsers are still supported by the
```

Comment:
Hmm, that feels like not the right place for this, since this section is related to the lib, not the CLI.

Maybe we can add this information to the `generate` documentation? Doesn't need to be a table, just noting the "min" and "max/default" versions. (I worry a bit about this bitrotting, but it'll probably caught quickly and be easy to fix.)
Then we can link this here (and remove the parenthetical remark in the previous paragraph)?